ups/meas_extensions_shapeHSM.cfg

```diff
@@ -3,7 +3,7 @@
 import lsst.sconsUtils
 
 dependencies = {
-    "required": ["utils", "afw", "meas_algorithms", "GalSim"],
+    "required": ["utils", "afw", "meas_base", "meas_algorithms", "GalSim"],
```

Comment:
Can you actually remove meas_algorithms here (and below, in the .table file)?  Or does this package still use it somehow?